pom.xml

```diff
@@ -6,7 +6,7 @@
 
     <groupId>com.evanlennick</groupId>
     <artifactId>retry4j</artifactId>
-    <version>0.15.0-SNAPSHOT</version>
+    <version>0.15.1-SNAPSHOT</version>
```

Comment:
you shouldn't modify this version, if we release to fix this it will be 0.15.0, not 0.15.1... the last release was 0.14.0

Reply:
Ok, got it, thanks